src/scores/probability/crps_impl.py

```diff
@@ -174,35 +174,39 @@ def crps_cdf(
     is used as an index for predictive CDF values. Various techniques are used to
     interpolate CDF values between indexed thresholds.
 
-    Given
-        - a predictive CDF `fcst` indexed at thresholds by variable x,
-        - an observation in CDF form `obs_cdf` (i.e., obs_cdf(x) = 0 if x < obs and 1 if x >= obs),
-        - a `threshold_weight` array indexed by variable x,
+    Given:
+        - a predictive CDF `fcst` indexed at thresholds by variable x
+        - an observation in CDF form `obs_cdf` (i.e., :math:`obs\\_cdf_x = 0` if :math:`x < obs` and 1 if :math:`x >= obs`)
+        - a `threshold_weight` array indexed by variable x
 
     The threshold-weighted CRPS is given by:
-        `CRPS = integral(threshold_weight(x) * (fcst(x) - obs_cdf(x))**2)`, over all thresholds x.
-    The usual CRPS is the threshold-weighted CRPS with `threshold_weight(x) = 1` for all x.
-    This can be decomposed into an over-forecast penalty
-        `integral(threshold_weight(x) * (fcst(x) - obs_cdf(x))**2)`, over all thresholds x where x >= obs
-    and an under-forecast penalty
-        `integral(threshold_weight(x) * (fcst(x) - obs_cdf(x))**2)`, over all thresholds x where x <= obs.
+        - :math:`twCRPS = \\int_{-\\infty}^{\\infty}{[\\text{threshold_weight}(x) \\times (\\text{fcst}(x) - \\text{obs_cdf}(x))^2]\\text{dx}}`, over all thresholds x.
```

Comment:
```suggestion
        - :math:`twCRPS = \\int_{-\\infty}^{\\infty}{[\\text{threshold_weight}(x) (\\text{fcst}(x) - \\text{obs_cdf}(x))^2]\\text{d}x}`, over all thresholds x.
```

Reply:
Happy to adjust the dx. Not sure why you have removed the times in just this one instance. Are you sure?